### PR TITLE
Fix opencacade solid shape

### PR DESCRIPTION
**Pull request by Luckabarbeau**
# Description of the problem

-Some steps could have more than one shell for one solid. In these cases, we were throwing an error. Now we simply use the solid.

# Description of the solution

-Now, we simply use the solid instead of throwing.

### CODE REVIEW

File: include/core/shape.h [hunk 1]
```diff
@@ -931,7 +931,15 @@ class OpenCascadeShape : public Shape<dim>
     // distance tool with just the shell.
     if (shells.size() > 0)
       {
-        if (shells.size() > 1)
+        if (solids.size() == 1)
```
Comment (bibeauv) [inline include/core/shape.h:934]: The first `if` statement uses the object `solids` while the second `if` statement uses the object `shells`. I just want to make sure that this is normal :)

**Review by bibeauv — COMMENTED** [review]
Bravo Lucka :p
Only thing, maybe just add a comment before the new `if` statement to explain what you do if the size is 1.
B

File: include/core/shape.h (hunk 1 above)
Comment (bibeauv) [inline include/core/shape.h:934]: It could be good to add a comment to explain what you do in the `if` statement!

Comment (blaisb) [reply include/core/shape.h:934]: Yeah clearly I would at least document this part.

**Review by blaisb — APPROVED** [review]

**Pull request merged**